docs/architecture/managers/chain-manager.md

```diff
@@ -39,20 +39,55 @@ The state of the actor is an instance of the [`ChainInfo`][chain] data structure
 /// ChainManager actor
 #[derive(Default)]
 pub struct ChainManager {
-    /// Blockchain information data structure
-    chain_info: Option<ChainInfo>,
-    /// Map that relates an epoch with the hashes of the blocks for that epoch
-    // One epoch can have more than one block
-    epoch_to_block_hash: HashMap<Epoch, HashSet<Hash>>,
-    /// Map that stores blocks by their hash
-    blocks: HashMap<Hash, Block>,
+    /// Blockchain state data structure
+    chain_state: ChainState,
     /// Current Epoch
     current_epoch: Option<Epoch>,
-    /// Block candidate to update chain_info in the next epoch
-    block_candidate: Option<Block>,
+    /// Transactions Pool (_mempool_)
+    transactions_pool: TransactionsPool,
+    /// Maximum weight each block can have
+    max_block_weight: u32,
+    /// Mining enabled
+    mining_enabled: bool,
+    /// Hash of the genesis block
+    genesis_block_hash: Hash,
+    /// Pool of active data requests
+    data_request_pool: DataRequestPool,
+    /// state of the state machine
+    sm_state: StateMachine,
+    /// Target beacon
```

Comment:
I think this chunk of code needs to be updated to what's been merged, which contains more accurate descriptions of some of these properties (specially `target_beacon`).